packages/babel-plugin-redux-saga/package.json

```diff
@@ -9,17 +9,20 @@
   ],
   "main": "./src/index",
   "scripts": {
-    "test": "node ./test/index.js | tap-difflet"
+    "test": "jest"
   },
   "devDependencies": {
     "@babel/core": "7.0.0-beta.39",
     "@babel/preset-env": "7.0.0-beta.39",
-    "tap-difflet": "^0.7.0",
-    "tape": "^4.8.0",
+    "babel-core": "^6.26.0",
```

Comment:
huh, funny that we do not have to fallback to more custom solution because babel got migrated to scoped packages